### PR TITLE
Auto-formatting

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
Second try at this, restarting from `master`. Simpler format file.

What do we think of this? Note that clang-format, unfortunately, can't insert braces around single-line conditionals. I.e., it can't change:
```
if (a) b();
```
to:
```
if (a) {
    b();
}
```

So we're just going to live without that. Aside from that, leave any other notes as to what you'd like to see.

cc @quinthar @flodnv @cead22

### CODE REVIEW

**Comment by tylerkaraszewski** [comment]
@cead22 - @quinthar and I discussed this and he's onboard with these mostly-default formatting settings. If you're happy let's merge!

**Comment by quinthar** [comment]
Yep, 👍

File: BedrockNode.cpp [hunk 1]
```diff
@@ -8,119 +8,102 @@
...
+void BedrockNode::postSelect(fd_map& fdm, uint64_t& nextActivity) {
     // Update the parent and attributes
-    SQLiteNode::postSelect( fdm, nextActivity );
+    SQLiteNode::postSelect(fdm, nextActivity);
 }
 
-bool BedrockNode::isReadOnly() {
-    return _readOnly;
-}
+bool BedrockNode::isReadOnly() { return _readOnly; }
```
Comment (flodnv) [inline BedrockNode.cpp]: this is not our style 😞

**Comment by quinthar** [comment]
Given that this is a standalone public repo, we opted to strip down the
style to match as near to the default clang as we could stomach.

**Pull request merged**